L/LMDB/build_tarballs.jl

```diff
@@ -5,17 +5,21 @@ version = v"0.9.25"
 
 # No sources, we're just building the testsuite
 sources = [
-    ArchiveSource("https://git.openldap.org/openldap/openldap/-/archive/LMDB_0.9.25/openldap-LMDB_$(version).tar.gz",
+    ArchiveSource("https://git.openldap.org/openldap/openldap/-/archive/LMDB_$(version)/openldap-LMDB_$(version).tar.gz",
                   "4f6eebe5ad98c10a75badd106f498ee2249d454352d048c78a49c99c940d4cae"),
 ]
 
 # Bash recipe for building across all platforms
-# rm: remove man files (it does not name sense)
-# exeext: Makefile does not support extensions - need to rename execuables manually
+# - CC: mdb_env_close0 segfaults if gcc is used instead of clang on Mac
```

Comment:
I'm not sure I understand this comment: the default compiler is already `clang` for macOS and FreeBSD

Reply:
`env` says `CC=cc` in the BinaryBuilder container for the MacOS target. I thought that was different from `clang`. Thanks to your comment, I've revisited this and confirmed that `cc` is in fact `clang`, so I can simplify the fix.

The root cause is [this line](https://git.openldap.org/openldap/openldap/-/blob/LMDB_0.9.25/libraries/liblmdb/Makefile#L21) which picks `gcc` for all targets (incl. MacOS). `make CC=$(CC) ...` should be enough to fix this.

Reply:
> `make CC=$(CC) ...` should be enough to fix this.

I think so, thanks!